gradle/runBootMicroserviceAcceptanceTests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -x
+
+function dumpCurrentMicroInfraSpringVersionToFile {
+
+    ./gradlew -q currentVersion | grep "Project version" | sed s/Project\ version\:\ //g > ~/.microInfraSpringCurrentVersion.txt
+    cat ~/.microInfraSpringCurrentVersion.txt
+}
+
+function cloneAndDoBuild {
+
+    echo Checking out and building $1 with micro-infra-spring `cat ~/.microInfraSpringCurrentVersion.txt`
+    git clone https://github.com/4finance/"$1".git
+    cd "$1"
+    echo "microInfraSpringVersion="`cat ~/.microInfraSpringCurrentVersion.txt` >> gradle.properties
```

Comment:
Unfotunately it turned out that `+` and `latest.integration` does not work with snapshot version released to local Maven repository. I asked on their [forum](http://forums.gradle.org/gradle/topics/latest-integration-does-not-resolve-the-latest-snapshot-version-from-mavenlocal).